tests/std/tests/P2502R2_generator/test.cpp

```diff
@@ -145,26 +95,26 @@ struct stateful_alloc {
     template <class U>
     constexpr stateful_alloc(const stateful_alloc<U>& that) noexcept : domain{that.domain} {}
 
-    T* allocate(const std::size_t n) {
+    T* allocate(const size_t n) {
         void* vp;
         if constexpr (alignof(T) > __STDCPP_DEFAULT_NEW_ALIGNMENT__) {
             vp = ::_aligned_malloc(n * sizeof(T), alignof(T));
         } else {
-            vp = std::malloc(n * sizeof(T));
+            vp = malloc(n * sizeof(T));
         }
 
         if (vp) {
             return static_cast<T*>(vp);
         }
 
-        throw std::bad_alloc{};
+        throw bad_alloc{};
     }
 
-    void deallocate(void* const vp, [[maybe_unused]] const std::size_t n) noexcept {
+    void deallocate(void* const vp, [[maybe_unused]] const size_t n) noexcept {

```

Comment:
Pre-existing, not worth resetting testing: `n` is always unused and this should just be a nameless `size_t`.